src/lib/tooltip/tooltip.spec.ts

```diff
@@ -80,7 +80,7 @@ describe('MatTooltip', () => {
     });
 
     it('should show and hide the tooltip', fakeAsync(() => {
-      expect(tooltipDirective._tooltipInstance).toBeUndefined();
+      expect(!!tooltipDirective._tooltipInstance).toBe(false);
```

Comment:
Can you make it just one place that does this? Something like a method `getTooltipInstance`? That way it's easy to add a comment in one place that explains it rather than repeating `!!` without explanation